README.md

```diff
@@ -10,44 +10,76 @@ required to deploy to different networks.
 Install ZeppelinOS, Ganache, and Truffle
 
 ```
-npm install -g truffle@5.0.2 ganache-cli@6.3.0 zos@2.2.0
+npm install -g truffle@5.0.2 ganache-cli@6.3.0 zos
```

Comment:
We should pin zos version because next version may easily break kits.

Reply:
Good point.

Reply:
Agreed!